doc/policies/authentication.rst

```diff
@@ -390,6 +391,20 @@ type: bool
 If a user authenticates successfully all failcounter of all of his tokens
 will be reset. This can be important, if using empty PINs or *otppin=None*.
 
+always_increase_failcounter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+type: bool
+
```

Comment:
Make clear, that it is about challenge response:

The normal behaviour is: to not increase the failcounter in case of challenge response.
If this policy is activated, the failcounter will be increased if the correct pin or password was given and a challenge was triggered.

Reply:
fixed in 4fe9f7f46e86b88dcd365672f86288d621345875.